web/libs/editor/tests/e2e/fragments/AtImageView.js

```diff
@@ -106,6 +110,7 @@ module.exports = {
 
   setZoom(scale, x, y) {
     I.executeScript(Helpers.setZoom, [scale, x, y]);
+    I.wait(1);
```

Comment:
```suggestion
    I.wait(2);
```
It looks like it may be not enough time to wait